core/store/src/main/java/mil/nga/giat/geowave/core/store/DataStore.java

```diff
@@ -73,4 +76,22 @@ public boolean delete(
 			final QueryOptions queryOptions,
 			final Query query );
 
+	/**
+	 * Public access to AdapterStore
+	 */
+	public AdapterStore getAdapterStore();
```

Comment:
I'll have to look at it. I must've added that because I needed access to the adapter store via the interface.